ios/Podfile

```diff
@@ -40,3 +48,16 @@ target 'SymbolWallet' do
   pod 'react-native-video', :path => '../node_modules/react-native-video'
 
 end
+
+def find_and_replace(dir, findstr, replacestr)
+  Dir[dir].each do |name|
+      text = File.read(name)
+      replace = text.gsub(findstr,replacestr)
+      if text != replace
+          puts "Fix: " + name
+          File.open(name, "w") { |file| file.puts replace }
+          STDOUT.flush
+      end
+  end
+  Dir[dir + '*/'].each(&method(:find_and_replace))
+end
```

Comment:
oh, I missed an end of line here, could you update @OlegMakarenko ?

Reply:
Fixed.